scripts/demo/template-projects/templates_in_master.json

```diff
@@ -0,0 +1,302 @@
+        "version": "2.10.0",
+        "label": "jupyter-base-notebook",
+        "inputs": {
+          "input_1": {
+            "nodeUuid": "template-uuid-4179-bc47-0905e6726758",
+            "output": "membrane_potential_json"
+          }
+        },
+        "inputNodes": [
+          "template-uuid-4179-bc47-0905e6726758"
+        ],
+        "outputNode": false,
+        "outputs": {},
+        "progress": 100,
+        "thumbnail": "https://opencor.ws/res/pics/logo.png",
+        "position": {
+          "x": 631,
+          "y": 259
+        }
+      }
+    }
+  },
+  {
+    "uuid": "template-uuid-5716-bedd-b409bb021760",
+    "name": "ISAN: 2D Plot",
+    "description": "2D RawGraphs viewer with one input",
+    "thumbnail": "https://user-images.githubusercontent.com/33152403/60168938-06a1bf00-9806-11e9-99ff-20c52d851add.png",
+    "prjOwner": "maiz@itis.swiss",
+    "creationDate": "2019-06-06 14:33:34.008",
+    "lastChangeDate": "2019-06-06 14:33:35.825",
+    "workbench": {
+      "template-uuid-58e5-884e-2690b3d54f11": {
+        "key": "simcore/services/frontend/file-picker",
+        "version": "1.0.0",
+        "label": "File Picker",
+        "inputs": {},
+        "inputNodes": [],
+        "outputNode": false,
+        "outputs": {
+          "outFile": {
+            "store": 1,
+            "path": "Shared Data/Height-Weight"
+          }
+        },
+        "progress": 100,
+        "thumbnail": "",
+        "position": {
+          "x": 100,
+          "y": 100
+        }
+      },
+      "template-uuid-58a5-83d7-57d0feffbfea": {
+        "key": "simcore/services/dynamic/raw-graphs",
+        "version": "2.8.0",
+        "label": "2D plot",
+        "inputs": {
+          "input_1": {
+            "nodeUuid": "template-uuid-58e5-884e-2690b3d54f11",
+            "output": "outFile"
+          }
+        },
+        "inputNodes": [
+          "template-uuid-58e5-884e-2690b3d54f11"
+        ],
+        "outputNode": false,
+        "outputs": {},
+        "progress": 90,
+        "thumbnail": "",
+        "position": {
+          "x": 400,
+          "y": 100
+        }
+      }
+    }
+  },
+  {
+    "uuid": "template-uuid-5d82-b08d-d39c436ca738",
+    "name": "ISAN: UCDavis use case: 0D",
+    "description": "Colleen Clancy Single Cell solver with a file picker and PostPro viewer",
+    "thumbnail": "https://user-images.githubusercontent.com/33152403/60168940-073a5580-9806-11e9-9a44-ae5266eeb020.png",
+    "prjOwner": "maiz@itis.swiss",
+    "creationDate": "2019-06-06 14:33:51.94",
+    "lastChangeDate": "2019-06-06 14:33:54.329",
+    "workbench": {
+      "template-uuid-59d6-b1a5-6e7b2773636b": {
+        "key": "simcore/services/frontend/file-picker",
+        "version": "1.0.0",
+        "label": "File Picker 0D",
+        "inputs": {},
+        "inputNodes": [],
+        "outputNode": false,
+        "outputs": {
+          "outFile": {
+            "store": 1,
+            "path": "Shared Data/initial_WStates"
```

Comment:
This needs to be fixed asap. I will create a corresponding Shared Folder on minio